src/test/java/org/kiwiproject/dropwizard/error/ErrorContextBuilderTest.java

```diff
@@ -130,6 +146,53 @@ private void assertIllegalArgumentExceptionThrownBuilding(SoftAssertions softly,
         }
     }
 
+    @Nested
+    class CleanupJob {
+
+        @Test
+        void shouldHaveCleanupApplicationErrorsJob() {
+            var cleanupConfig = new CleanupConfig();
+            cleanupConfig.setCleanupJobName("foo-%d");
+
+            var builder = ErrorContextBuilder.newInstance()
+                    .environment(environment)
+                    .serviceDetails(serviceDetails)
+                    .cleanupConfig(cleanupConfig)
+                    .dataStoreType(DataStoreType.NOT_SHARED);
+
+            builder.buildInMemoryH2();
+
+            var dataSourceFactory = ApplicationErrorJdbc.createInMemoryH2Database();
+            builder.buildWithDataStoreFactory(dataSourceFactory);
+
+            var jdbi = Jdbi3Builders.buildManagedJdbi(environment, dataSourceFactory);
+             builder.buildWithJdbi3(jdbi);
+
+             verify(executor, times(3))
+                     .scheduleWithFixedDelay(any(CleanupApplicationErrorsJob.class), eq(1L),
+                             eq(Duration.days(1).toMinutes()), eq(TimeUnit.MINUTES));
+        }
+
+        @Test
+        void shouldAllowSkippingCleanupJobRegistration(SoftAssertions softly) {
```

Comment:
`softly` is never used in this test. Did you intend to use it, or should the argument be removed?